docs/fault-proof-alpha/manual.md

```diff
@@ -0,0 +1,110 @@
+## Manual Fault Proof Interactions
+
+## Creating a Game
+
+The process of disputing an output root starts by creating a new dispute game. There are conceptually three key inputs
+required for a dispute game:
+
+- The output root being disputed
+- The agreed output root the derivation process will start from
+- The L1 head block that defines the canonical L1 chain containing all required batch data to perform the derivation
+
+The creator of the game selects the output root to dispute. It is identified by its L2 block number which can be used to
+look up the full details in the L2 output oracle.
+
+The agreed output root is defined as the output root immediately prior to the disputed output root in the L2 output
+oracle. Therefore, a dispute game should only be created for the first invalid output root. If it is successfully
+disputed, all output roots after it would be considered invalid by inference.
```

Comment:
```suggestion
oracle. Therefore, it only makes sense to crate a dispute game for the first invalid output root. If it is successfully
disputed, all output roots after it are considered invalid by inference.
```

Reply:
I don't think saying "only makes sense" is strong enough here.  If you dispute output roots after the first invalid output root, you will likely lose because the agreed starting point is invalid but you've just incorrectly agreed to it.

Reply:
Right, I was moreso tuning the language towards making it clear that it is _possible_ but non sensical. I think your language actually reads close to this, so no need to change :)